Documentation/auth_api.md

```diff
@@ -229,27 +334,48 @@ GET/HEAD  /v2/auth/roles
           "roles": ["fleet", "etcd", "quay"]
         }
 
-**Get Role Details**
```

Comment:
It didn't get removed. I moved it up to be before the **Get a list of Roles** section. I thought it would be good to have the **Get a list of Roles with details** come right after the **Get a list of Roles** section to basically group the similar requests next to each other.